portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,62 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+// import com.google.sps.data.ServerStats;
+import com.google.gson.Gson;
+
+/** Servlet that handles comments data.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private ArrayList<String> comments;
+    static boolean initialized;
+
+    static {
```

Comment:
I have never seen this syntax before.

"3) Java static block" -> You learn something new everyday

Reply:
:D . I am glad I could help